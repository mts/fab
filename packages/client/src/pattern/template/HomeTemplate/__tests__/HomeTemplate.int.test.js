import React from 'react'
import { defaultRender } from './HomeTemplate.int.render'

jest.mock('react-router-dom', () => ({ Link: () => <div>some link</div> }))

describe('<HomeTemplate />', () => {
  describe('Snaphot', () => {
    test('must match defaultRender', () => {
      expect(global.renderToJSON(defaultRender)).toMatchSnapshot()
    })
  })
})
