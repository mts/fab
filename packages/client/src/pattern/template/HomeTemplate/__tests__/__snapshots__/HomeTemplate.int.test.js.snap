// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeTemplate /> Snaphot must match defaultRender 1`] = `
<div
  className="homeTemplate"
>
  HomeTemplate under construction
</div>
`;
